gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -277,6 +305,34 @@ def filter_depth(
 
             # Combine geometric mask and confidence mask
             joint_mask = np.logical_and(confidence_mask, geo_mask)
+
+            # Compute reprojection error after filtering by joint mask
+            height, width = ref_depth_est.shape[:2]
```

Comment:
let's move lines 310-334 to a helper function that just computes the reprojection errors for us.

Reply:
yeah, and preferably unit test it